osquery/distributed/distributed.cpp

```diff
@@ -158,7 +205,7 @@ Status Distributed::runQueries() {
     // Keep track of the currently executing request
     Distributed::setCurrentRequestId(request.id);
 
-    SQL sql(request.query);
+    auto sql = monitorNonnumeric(request.id, request.query);
```

Comment:
Any reason to use `auto` here rather than `SQL`? I always find `auto` convenient to write, but harder to read.

Reply:
Everything seems to use auto in this file so I wanted to match code style.